### PR TITLE
feat: added prompt option for ask command

### PR DESCRIPTION
**Pull request by NguyenQuang-10**
This PR resolves https://github.com/bifrostlab/llm-assistant/issues/22

### CODE REVIEW

**Review by nqngo — APPROVED** [review]
LGTM

File: discord_bot/bot.py [hunk 1]
```diff
@@ -37,8 +37,11 @@ async def my_long_command_function(ctx: interactions.SlashContext):
   opt_type=interactions.OptionType.STRING,
   autocomplete=True,
 )
-async def ask_model(ctx: interactions.SlashContext, model: str):
-  await ctx.send(f"You asked model {model}")
+@interactions.slash_option(
+  name="prompt", description="Enter your prompt", required=True, opt_type=interactions.OptionType.STRING
```
Comment (nqngo) [inline discord_bot/bot.py:41]: Maybe specify a sensible min_length to avoid wasting an API call on accidentally send off an incomplete prompt

Comment (rodonguyen) [reply discord_bot/bot.py:41]: make it >=10 chars?

File: discord_bot/bot.py
Comment (keduong33) [inline discord_bot/bot.py:1]: My ruff auto-organise the imports -> I assume that is the right format?

**Review by nqngo — APPROVED** [review]

File: discord_bot/bot.py
Comment (rodonguyen) [reply discord_bot/bot.py:1]: I think it's ok :)

**Pull request merged**